e2etest/pkg/k8s/nodes.go

```diff
@@ -59,6 +59,51 @@ func NodeIPsForFamily(nodes []v1.Node, family ipfamily.Family, vrfName string) (
 	return res, nil
 }
 
+// NodeIPsForExcludeBalancers return the IP addresses of all nodes with the label
+// 'node.kubernetes.io/exclude-from-external-load-balancers'
+func NodeIPsForExcludeBalancers(nodes []v1.Node, family ipfamily.Family, vrfName string) ([]string, error) {
```

Comment:
As above, it'd be simpler to just filter out the node as a whole (or am I missing something?)

Reply:
please forget it... :(